src/Microsoft.Azure.SignalR.Management/Configuration/ServiceManagerOptions.cs

```diff
@@ -11,6 +12,11 @@ namespace Microsoft.Azure.SignalR.Management
     /// </summary>
     public class ServiceManagerOptions
     {
+        /// <summary>
+        /// The section name used to get <see cref="ServiceManagerOptions"/> value from <see cref="IConfiguration.GetSection(string)"/>
+        /// </summary>
+        public const string Section = "Azure:SignalR";
```

Comment:
intended to expose it to customer?

Reply:
Yes, I think customer can get the right position of configuration from `Section`.